Formula/lensfun.rb

```diff
@@ -3,25 +3,14 @@ class Lensfun < Formula
 
   desc "Remove defects from digital images"
   homepage "https://lensfun.github.io/"
-  url "https://downloads.sourceforge.net/project/lensfun/0.3.95/lensfun-0.3.95.tar.gz"
-  sha256 "82c29c833c1604c48ca3ab8a35e86b7189b8effac1b1476095c0529afb702808"
+  url "https://github.com/lensfun/lensfun/archive/refs/tags/v0.3.3.tar.gz"
+  sha256 "57ba5a0377f24948972339e18be946af12eda22b7c707eb0ddd26586370f6765"
   license all_of: [
     "LGPL-3.0-only",
     "GPL-3.0-only",
     "CC-BY-3.0",
     :public_domain,
   ]
-  revision 4
-
-  bottle do
-    sha256 arm64_monterey: "530ebafb7cb54daaa3095f543ba8f05e331fd8a36265fbb2cfbe482e5822a223"
```

Comment:
No need to touch this block.

Reply:
Thanks @SMillerDev, please excuse, my understanding is that 
- as it is the initial revision of a new package/project version (first to provide lensfun 0.3.3; revision 4 was of the former lensfun 0.3.95) the revision number needs to be removed. `revision n` would than be an iteration within the same package/project version to adjust build flags or apply patches to the default.
- as the bottles will be built based on a different package/project version, any checksum will be invalid and I understood that those will be added automatically?

I am not expert. So you are basically saying that I need to restore those lines, correct?

Reply:
The bottle sections are back. Should I also add the revision? What revision number to take?

Reply:
No, the bottles are managed by CI. Removing the bottle block won't have any affect.

The revision should be removed because that works as you expected.

Reply:
Perfect, thanks @SMillerDev for your quick answer.